code/modules/projectiles/guns/misc/beam_rifle.dm

```diff
@@ -183,6 +183,7 @@
 /obj/item/gun/energy/beam_rifle/Initialize()
 	. = ..()
 	current_tracers = list()
+	flags_2 |= SLOWS_WHILE_IN_HAND_2

```

Comment:
??

Reply:
SLOWS WHILE IN HAND

Reply:
oh yeah beam rifles are supposed to slow the user when aiming but it didn't do that for some reason this fixes that ayyy

Reply:
oh ok can you update CL?